corehq/apps/locations/templates/locations/manage/location.html

```diff
@@ -4,14 +4,27 @@
 {% load hqstyle_tags %}
 {% load crispy_forms_tags %}
 
+{% block head %} {{ block.super }}
```

Comment:
most of the changes in this file have just been moved from the now defunct `locations/manage/products_per_location.html`.